src/01-practices/codyMan0/2주차/cartItem/CartItem.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { CartItem } from './CartItem';
+import { FoodInfoType } from '../type';
+
+describe('CardItem Component', () => {
+  const foodInfoWithoutRecommendation: FoodInfoType = {
+    title: '물떡 & 어묵탕',
+    subTitle: '칼칼한 국물과 탱탱한 밀가래떡과 어묵',
+    price: 4900,
+    img: './asset/img.jpeg',
+    recommendOfOwner: false,
+  };
+
+  test('renders price with priceDetail when priceDetail is present', () => {
+    render(<CartItem foodInfo={foodInfoWithoutRecommendation} />);
+
+    const badgeElement = screen.queryByText('사장님 추천');
+
+    // recommendOfOwner가 false인 경우, badgeElement가 null임을 확인
+    expect(badgeElement).toBeNull();
```

Comment:
DOM에 없는 지 체크하는 게 더 알맞을 것 같아요
```suggestion
    expect(badgeElement).not.toBeInTheDocument();
```

Reply:
아하! 넵!